kratos/includes/enums.h

```diff
@@ -0,0 +1,59 @@
+//    |  /           |
```

Comment:
ouch, should this be that global? i'll accept that for now, however i wonder if we should make it that public.

@roigcarlo @pooyan-dadvand opinions?